great_expectations/execution_engine/sqlalchemy_execution_engine.py

```diff
@@ -257,6 +257,13 @@ def __init__(
                     "Credentials or an engine are required for a SqlAlchemyExecutionEngine."
                 )
 
+        # these are two backends where temp_table_creation is not supported we set the default value to False.
+        if self.engine.dialect.name.lower() in [
+            "trino",
```

Comment:
Hi @alexsherstinsky we currently don't allow users of the `trino` backend to use `batch_spec_passthrough` to override the default value of `create_temp_table`, because of the following line in [get_batch_data_and_markers()](https://github.com/great-expectations/great_expectations/blob/a70c76917ffa4bca4b60a8575836030e86d1e276/great_expectations/execution_engine/sqlalchemy_execution_engine.py#L1100)

Setting the default in the `__init__` will allow for an override if necessary (although very unlikely). What are your thoughts on moving the logic up to the `__init__` for `SqlAlchemyExecutionEngine`?

Reply:
@Shinnnyshinshin Great idea -- thanks!